examples/data-block-examples/landsat8-geotiff-example.ipynb

```diff
@@ -0,0 +1,326 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "straight-candy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/wilhelmina.nekoto/.pyenv/versions/UP42-py-dev/lib/python3.7/site-packages/pandas/compat/__init__.py:97: UserWarning: Could not import the lzma module. Your installed Python is incomplete. Attempting to use lzma compression will result in a RuntimeError.\n",
+      "  warnings.warn(msg)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import up42"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "occasional-north",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 11:51:31,084 - Authentication with UP42 successful!\n",
+      "2021-03-15 11:51:32,176 - Initialized Project(name: Landsat8-data-example, project_id: fbac95e6-1d78-4953-9485-b6fc072cb4c2, description: , createdAt: 2021-03-15T10:48:54.591911Z)\n"
+     ]
+    }
+   ],
+   "source": [
+    "up42.authenticate(project_id=\"fbac95e6-1d78-4953-9485-b6fc072cb4c2\", \n",
+    "                  project_api_key=\"NzrlHj9r.tjwvGhLRmjtlmtoBhSZlWdTE1IAc0ATfs8b\")\n",
+    "project = up42.initialize_project()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "geographic-armor",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 11:54:52,672 - Getting small example aoi in location 'Berlin'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "aoi = up42.get_example_aoi(as_dataframe=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "refined-estonia",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2021-03-15 12:01:53,155 - Getting existing workflows in project ...\n",
+      "2021-03-15 12:01:53,740 - Got 1 workflows for project fbac95e6-1d78-4953-9485-b6fc072cb4c2.\n",
+      "100%|██████████| 1/1 [00:00<00:00,  1.66it/s]\n",
+      "2021-03-15 12:01:54,344 - Using existing workflow: Landsat8-Geotiff-example - 2b4b6d62-2207-449c-a46d-658d5f960032\n",
+      "2021-03-15 12:01:58,264 - Added tasks to workflow: [{'name': 'sentinelhub-landsat8-aoiclipped:1', 'parentName': None, 'blockId': '95519b2d-09d7-4cd0-a321-4d6a46bef6c1'}]\n"
```

Comment:
Unfortunately you picked the wrong data block. I believe it should be this one, but best to confirm with @nb-42